source_gen/lib/src/utils.dart

```diff
@@ -2,10 +2,13 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
Am I blind? Where is `dart:io` used here?

Reply:
Its used in the initialization closure for the final variable `rootPackageName` :)